src/main/java/edu/indiana/dlib/amppd/config/JwtRequestFilter.java

```diff
@@ -24,43 +25,89 @@
 	@Autowired
 	private AmpUserService jwtUserDetailsService;
 	
+	
 	@Autowired
 	private JwtTokenUtil jwtTokenUtil;
 	
+	@Autowired
+	private AmppdUiPropertyConfig amppdUIConfig;
+	
+
+	@Autowired
+	private AuthService authService;
+	
+	
+	private void CreateAnonymousAuth(HttpServletRequest request) {
+		AmpUser userDetails = new AmpUser();
+		userDetails.setEmail("none");
+		userDetails.setUsername("");
+	
+		UsernamePasswordAuthenticationToken usernamePasswordAuthenticationToken = new UsernamePasswordAuthenticationToken(
+				userDetails, null, null);
+
+		usernamePasswordAuthenticationToken.setDetails(new WebAuthenticationDetailsSource().buildDetails(request));
+
+		SecurityContextHolder.getContext().setAuthentication(usernamePasswordAuthenticationToken);
+	}
+	private boolean ValidRefUrl(String referer) {
+		String cleanedRef = referer.replace("https://", "").replace("#/", "").replace("#", "").replace("localhost", "127.0.0.1");
+		String cleanedUiUrl = amppdUIConfig.getUrl().replace("https://", "").replace("#/", "").replace("#", "").replace("localhost", "127.0.0.1") + "timeliner.html";
+		logger.debug(cleanedUiUrl + " starts with " + cleanedRef + " : " + cleanedUiUrl.startsWith(cleanedRef));
```

Comment:
aren't we using http not https now? would be nice if there is a way to define timeliner url as property and use regexp to matching the url pattern?